projects/ontimize-web-ngx/src/lib/components/input/listpicker/o-list-picker.component.ts

```diff
@@ -10,7 +10,7 @@ import {
   OnInit,
   Optional,
   SimpleChange,
-  ViewChild
+  ViewChild,
```

Comment:
can this trailing comma cause tslint problems?

Reply:
I dont think so, but I had the typescript hero plugin misconfigured and it was adding this comma.